R/logging.r

```diff
@@ -7,19 +7,53 @@
 stderror <- stderr()
 
 log_debug <- function(...) {
-    if (getOption('jupyter.log_level') >= 3L) {
-        cat('DEBUG: ', sprintf(...), '\n', sep = '', file = stderror)
+    if (isTRUE(getOption('jupyter.log_level') >= 3L)) {
+        log_msg_stderror('DEBUG', sprintf(...))
+        log_msg_logfile('DEBUG', sprintf(...))
```

Comment:
tried that, but then wanted to send the filename to the normal log and didn't want to get into recursion and so split both calls...